src/libs/ReportUtils.ts

```diff
@@ -457,6 +458,19 @@ Onyx.connect({
     callback: (value) => (allPolicies = value),
 });
 
+let allPolicyReportFields: OnyxCollection<PolicyReportFields>;
```

Comment:
I hope it's not one of these collections we're nuking and un-nuking all over again...

Reply:
Yeah, it was a general comment, not a change request. This one doesn't need a resolution.